wolfcrypt/src/asn.c

```diff
@@ -14315,7 +14322,13 @@ static int GetHashId(const byte* id, int length, byte* hash, int hashAlg)
 /* Mapping of certificate name component to useful information. */
 typedef struct CertNameData {
     /* Type string of name component. */
+#ifdef WOLFSSL_NAMES_STATIC
+    const char str[20];
```

Comment:
Maybe a comment that this is large enough for the strings below, or explaining why 20.

Reply:
done